src/main/resources/assets/cctweaks/lua/prebios-string.lua

```diff
@@ -71,4 +87,12 @@ if native_getfenv then
 	end
 end
 
-return fun()
+-- Try to run the bios, print possible errors
+local _, ok, err = pcall(fun)
```

Comment:
Well I don't know if pcall for example still returns false if the file can successfully load but returns for example "attempt to index nil" or something like that